src/main/java/org/mskcc/cbio/portal/util/CaseListReader.java

```diff
@@ -0,0 +1,43 @@
+package org.mskcc.cbio.portal.util;
+
+import org.mskcc.cbio.portal.scripts.TrimmedProperties;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class CaseListReader {
+
+    public static CaseList readFile(File caseListFile) {
+        Properties properties = new TrimmedProperties();
+        try {
+            properties.load(new FileReader(caseListFile));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        String stableId = properties.getProperty("stable_id");
+        String cancerStudyIdentifier = properties.getProperty("cancer_study_identifier");
+        String caseListName = properties.getProperty("case_list_name");
+        String caseListDescription = properties.getProperty("case_list_description");
+        String caseListCategory = properties.getProperty("case_list_category");
+        String caseListIds = properties.getProperty("case_list_ids");
+        List<String> sampleIds = caseListIds == null ? List.of()
```

Comment:
I think case_list_ids is mandatory, so we should throw error if null

Reply:
It's done in [CaseListValidator](https://github.com/cBioPortal/cbioportal-core/blob/rfc79/src/main/java/org/mskcc/cbio/portal/validate/CaseListValidator.java#L24). The reader is not concerned with what fields are compulsory in the current design.